elementary/monitor/api/filters/filters.py

```diff
@@ -20,8 +20,8 @@
 class FiltersAPI(APIClient):
     def get_filters(
         self,
-        test_results_totals: Dict[Optional[str], TotalsSchema],
```

Comment:
keys of test_results, test_runs, test_results_totals, test_runs_totals are no longer nullable